main.go

```diff
@@ -41,6 +42,8 @@ type GlobalFlags struct {
 	URI                   string `name:"mongodb.uri" help:"MongoDB connection URI" env:"MONGODB_URI" placeholder:"mongodb://user:pass@127.0.0.1:27017/admin?ssl=true"`
 	GlobalConnPool        bool   `name:"mongodb.global-conn-pool" help:"Use global connection pool instead of creating new pool for each http request."`
 	DirectConnect         bool   `name:"mongodb.direct-connect" help:"Whether or not a direct connect should be made. Direct connections are not valid if multiple hosts are specified or an SRV URI is used." default:"false"`
+	AuthMechanism         string `name:"mongodb.auth-mechanism" help:"Authentication mechanism. Example: MONGODB-X509."`
```

Comment:
Why don't pass Authmechanism as part of DSN?

Reply:
As far as I see mongo-driver knows how to parse it.

Reply:
AFAIK we should support two ways flag and URI: https://jira.percona.com/browse/PMM-7278?focusedCommentId=277074&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#/comment-277074